src/masonite/drivers/broadcast/BroadcastAblyDriver.py

```diff
@@ -52,8 +52,9 @@ def channel(self, channels, message, event='base-event'):
             raise DriverLibraryNotFound(
                 'Could not find the "ably" library. Please pip install this library running "pip install ably"')
 
+        configuration = config('broadcast.drivers.ably')
         client = AblyRest('{0}'.format(
-            self.config.DRIVERS['ably']['secret']
+            configuration['secret']
```

Comment:
This won't work. If the ably config doesn't exist then configuration will return `None`

Reply:
@josephmancuso 

Will it work?

```suggestion
            configuration['secret'] or []
```

Reply:
No that won't work. configuration would be None and you can't get `['secret']` from None

Reply:
I beleive `config` takes a second parameter which is a default argument. So maybe `config('dot.notation.here', '')` would work

Reply:
It should probably throw an error though instead since you can't even use it without a secret set

Reply:
BroadcastPushDriver has similar code [here](https://github.com/MasoniteFramework/masonite/blob/2.3/src/masonite/drivers/broadcast/BroadcastPusherDriver.py#L54)

Either both work (cause configuration is always available) or both need to be fixed IMO

Reply:
So whats the conclusion, can I just remove the unused app parameter in init, and raise the PR again?

If anything else required let me know!

Reply:
Let @josephmancuso comment.